pylint/checkers/match_statements_checker.py

```diff
@@ -46,6 +63,19 @@ class MatchStatementChecker(BaseChecker):
             "Emitted when there is more than one sub-pattern for a specific "
             "attribute in a class pattern.",
         ),
+        "R1905": (
+            "Use '%s() as %s' instead",
```

Comment:
```suggestion
            "Use '%s() as %s' instead, if performance is a concern",
```
(This is a suggestion, the message is short, there's room for explanation in it, maybe ?)